frontend/src/components/pages/Scheduling/ConfirmDetails.tsx

```diff
@@ -104,7 +104,6 @@ const ConfirmDetails = ({
   };
```

Comment:
there's a bunch of unused imports in this file that should be removed
weird that it should be manual, thought our linter had the eslint rule: https://www.npmjs.com/package/eslint-plugin-unused-imports